ruby/GETTING_STARTED.md

```diff
@@ -0,0 +1,23 @@
+# Extism
+
+Getting started
+
+```ruby
+require "/extism"
```

Comment:
should this be `extism` without the leading slash? for end-users expecting to get it from rubygems?